### PR TITLE
[Workers] Github notifs tutorial revision

### PR DESCRIPTION
**Pull request by deadlypants1973**
do not merge

### CODE REVIEW

File: products/workers/src/content/tutorials/github-sms-notifications-using-twilio/index.md [hunk 1]
```diff
@@ -91,7 +91,7 @@ async function handleRequest(request) {
 }
 ```
 
-Let’s start by modifying the starter code to handle a `POST` response and renaming the request handler. We can use the `request.method` property of [request](/runtime-apis/request) to check if the request is a `POST` request, and send an error response if incorrect. The `simpleResponse` function is an easy wrapper for you to respond with requests using your Worker.
+Begin by modifying the starter code to handle a `POST` response and renaming the request handler. You can use the `request.method` property of [request](/runtime-apis/request) to check if the request is a `POST` request, and send an error response if incorrect. The `simpleResponse` function is an easy wrapper for you to respond with requests using your Worker.
```
Comment (deadlypants1973) [inline products/workers/src/content/tutorials/github-sms-notifications-using-twilio/index.md]: @lukeed i feel like this is phrased kinda strangely but lmk what u think i didn't modify it

Comment (lukeed) [reply products/workers/src/content/tutorials/github-sms-notifications-using-twilio/index.md]: You can check for incoming POST requests by inspecting the `request.method` property and reply with an error response if it isn't.

Comment (deadlypants1973) [inline products/workers/src/content/tutorials/github-sms-notifications-using-twilio/index.md]:
```suggestion
Begin by modifying the starter code to handle a `POST` response and renaming the request handler. You can use the `request.method` property of [request](/runtime-apis/request) to check if the request is a `POST` request, and send an error response if the request is not a `POST` request. The `simpleResponse` function is an easy wrapper for you to respond with requests using your Worker.
```

**Review by lukeed — APPROVED** [review]

File: products/workers/src/content/tutorials/github-sms-notifications-using-twilio/index.md [hunk 2]
```diff
@@ -36,39 +36,39 @@ $ wrangler generate github-twilio-notifications
...
 ## Configure GitHub
 
-The first step is to configure a GitHub webhook to post to our Worker when there is an update to the repo.
+To start, you will need to configure a GitHub Webhook to post to your Worker when there is an update to the repository:
 
-Click on the "Settings" tab for your repo and then click on the "Webhooks" page from the list on the left.
+1. Go to your Github repository's **Settings** > **Webhooks** > **Add webhook**.
 
-Click "Add webhook" and configure the "Payload URL" with your worker URL. You can find your worker URL by setting your account id in the `wrangler.toml` file and then running "wrangler publish" in your command line to generate a live URL for your Worker.
+2. Set the Payload URL to the `/webhook` path on your Worker URL. . You can find your worker URL by populating [your account id in the `wrangler.toml`](/get-started/guide#6-preview-your-project) file and then [running `wrangler publish` in your command line](/get-started/guide#8-publish-your-project) to generate a live URL for your Worker.
```
Comment (marciocloudflare) [inline products/workers/src/content/tutorials/github-sms-notifications-using-twilio/index.md]:
```suggestion
2. Set the Payload URL to the `/webhook` path on your Worker URL. You can find your worker URL by populating [your account id in the `wrangler.toml`](/get-started/guide#6-preview-your-project) file and then [running `wrangler publish` in your command line](/get-started/guide#8-publish-your-project) to generate a live URL for your Worker.
```

File: products/workers/src/content/tutorials/github-sms-notifications-using-twilio/index.md [hunk 3]
```diff
@@ -36,39 +36,39 @@ $ wrangler generate github-twilio-notifications
...
 
-For the "Content type", choose "application/json".
+3. In the **Content type** dropdown, select **application/json**.
```
Comment (marciocloudflare) [inline products/workers/src/content/tutorials/github-sms-notifications-using-twilio/index.md]:
```suggestion
3. In the **Content type** dropdown, select _application/json_.
```

Comment (marciocloudflare) [reply products/workers/src/content/tutorials/github-sms-notifications-using-twilio/index.md]: As this is a dropdown, it should be italicized according to our style guide.

File: products/workers/src/content/tutorials/github-sms-notifications-using-twilio/index.md [hunk 4]
```diff
@@ -36,39 +36,39 @@ $ wrangler generate github-twilio-notifications
...
 
 ![GitHub config screenshot](media/github-config-screenshot.png)
 
 --------------------------------
 
-## Parsing the Response
+## Parsing the response
 
-With your local environment setup, we will now parse the repo update with your worker. If you get stuck, you can refer to the finished [index.js](https://github.com/davidtsong/GitHub-twilio-notifications/blob/master/index.js).
+With your local environment setup, you will now parse the repository update with your Worker. If you experience any difficulty, refer to the finished [index.js](https://github.com/davidtsong/GitHub-twilio-notifications/blob/master/index.js).
```
Comment (marciocloudflare) [inline products/workers/src/content/tutorials/github-sms-notifications-using-twilio/index.md]:
```suggestion
With your local environment set up, you will now parse the repository update with your Worker. If you experience any difficulty, refer to the finished [index.js](https://github.com/davidtsong/GitHub-twilio-notifications/blob/master/index.js).
```

**Review by marciocloudflare — APPROVED** [review]

**Pull request merged**